doc/index.qdoc

```diff
@@ -115,34 +116,143 @@ WebView currently supports customization via the following interfaces:
 \endlist
 
+\section2 Debugging Web Applications
+
+When debugging your application you can run it with the \c EMBED_CONSOLE=1
+environment variable. In case you are interested in the requests that are
+made use the \c EMBED_CONSOLE=network environment variable. When
+\c EMBED_CONSOLE is enabled console messages are printed to the terminal.
 
 \section2 Loading and using local resources
 
 When \l {https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS}{Cross-Origin Resource Sharing}
-is applied resources that are required and loaded from another origin should be made
-aware of the cross-origin boundary. It is preferable that resources are served from a web service instead of using
-local resources. When resources are served from a web service you can specify
-appropriate HTTP-headers to permit loading of resources. Content security manager
-will intercept and verify requests that you are performing and in case there is a violation
-the request is denied.
-
-In case one doesn't have the possibility to setup a web service and there is need
-to load resources with file uri scheme. You can set \c security.disable_cors_checks
-preference to \c true to bypass cross-origin resource sharing checks. This
-preference should be set before loading of any resources and only if you know what
-requests are sent.
-
-When debugging your application you can run it with the \c EMBED_CONSOLE=1 environment variable. In case
-you are interested in the requests that are made use the \c EMBED_CONSOLE=network environment
-variable. When \c EMBED_CONSOLE is enabled console messages are printed to the terminal.
+is applied resources that are required and loaded from another origin
+should be made aware of the cross-origin boundary. The content security
+manager will intercept and verify requests that you are performing and in
+case there is a violation the request is denied.
 
-Example of failure could look like this:
+With debugging enabled as described above, an example CORS failure could
+look like this:
 
 \code
 [JavaScript Error: "Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote
 resource at file:///path-to-resource/myfile.js. (Reason: CORS request not http)."]
 \endcode
 
+In particular, this is likely to block files served from local storage
+using the file URI scheme. In case you're developing a Web App with the
+need to serve local files, there are several approaches you can use to
+circumnavigate this.
+
+\section3 Option 1. Don't serve the files locally at all
+
+In many cases, while it can initially seem sensible to serve local files,
+you may find you can get the same or better capabilities by serving a
+remote version of your app instead.
+
+Many Web Apps will use caching, \l {https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API}{Web Storage}
+and similar technologies so that they can operate fully even when your
+phone is offline. The app will be served from the remote server on first
```

Comment:
so that they can operate fully even when offline.

Simpler not to mention "phone".

Reply:
Yeah, that's better. Thank you.